### PR TITLE
Start release notes for 15.5

### PR DESCRIPTION
**Pull request by rainersigwald**
It'll be easier to keep this up to date than to reconstruct significant changes from `git log` (which is what I did so far, but there haven't been too many changes in master yet).

### CODE REVIEW

**Comment by AndyGerlicher** [comment]
Should we put this in `/documentation/ReleaseNotes.md`?

**Comment by magol** [comment]
Have you considered keepachangelog.com?

**Comment by rainersigwald** [comment]
@magol I hadn't seen that, thanks!

File: documentation/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+This document forms the basis of release notes for MSBuild.
+
+MSBuild does **not** use Semantic Versioning.
```
Comment (magol) [inline documentation/CHANGELOG.md]: It is strange to mention what the project DOES NOT follow Semantic Versioning. Why not remove this note?

Why not change this according to the [specification](http://keepachangelog.com/en/1.0.0/) (but remove the irrelevant about Semantic Versioning):

> All notable changes to this project will be documented in this file.
> 
> The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).

**Comment by magol** [comment]
Really good start.
Is it also possible to add the change logs for earlier versions to? At least some versions back.

**Comment by magol** [comment]
@AndyGerlicher @rainersigwald
What a shame that this has not become reality.
Will the change log be handled in another way?

**Comment by AndyGerlicher** [comment]
It was something we wanted to do but haven't had a chance to prioritize and we wanted to reduce open PRs. We plan to start a new one and would incorporate this into that.

**Pull request closed without merging**